nextjs/pages/api/random-pixels.js

```diff
@@ -0,0 +1,24 @@
+import seedrandom from "seedrandom";
+import promClient, { register } from "./prom-client";
+
+const httpRequestTimer = new promClient.Histogram({
+  name: "nextjs_random_api_response_time",
```

Comment:
Prometheus has its own way of naming metrics. How about something like:
```suggestion
  name: "nextjs_api_random_duration_seconds",
```